### ISSUE TITLE
Remove 'awaiting ...' labels upon merge.

### ISSUE THREAD

**Issue by terryjreedy** [python/bedevere#53]
Merging PR with 'awaiting merge' or 'awaiting review' does not remove labels.

**Comment by brettcannon** [comment]
Nope, I didn't code anything up to remove labels (for no specific reason, mind you).
